stories/core/Carousel.stories.tsx

```diff
@@ -105,3 +110,99 @@ Controlled.args = {};
 Controlled.argTypes = {
   activeSlideIndex: { control: { disable: true } },
 };
+
+const useId = () => 'my-custom-carousel';
+export const Custom: Story<CarouselProps> = () => {
+  const gradients = [
+    { from: '#cc2b5e', to: '#753a88' },
+    { from: '#00467f', to: '#a5cc82' },
+    { from: '#2193b0', to: '#6dd5ed' },
+    { from: '#ffe000', to: '#799f0c' },
+    { from: '#e65c00', to: '#f9d423' },
+    { from: '#1488cc', to: '#2b32b2' },
+    { from: '#bbd2c5', to: '#536976' },
+    { from: '#9796f0', to: '#fbc7d4' },
+    { from: '#b79891', to: '#94716b' },
+    { from: '#acb6e5', to: '#86fde8' },
+  ];
+
+  const id = useId();
```

Comment:
Why you are hiding that id :DDD

Reply:
thought we could just use the new `useId` when we upgrade to react 18